client/src/js/components/Fido/useFido.jsx

```diff
@@ -73,170 +71,96 @@ function fetchStateReducer(state, action) {
     case ACTIONS.initiateFetch:
       return {
         ...state,
-        [payload.name]: {
-          inFlight: true,
-          success: false,
-          fail: false,
-          data: payload.preserveData ? state.data : null,
-          error: null,
-        },
+        inFlight: true,
+        success: false,
+        fail: false,
+        data: payload.preserveData ? state.data : null,
+        error: null,
       };
     case ACTIONS.setData:
       return {
         ...state,
-        [payload.name]: {
-          inFlight: false,
-          success: true,
-          fail: false,
-          data: payload.json,
-          error: null,
-        },
+        inFlight: false,
+        success: true,
+        fail: false,
+        data: payload.json,
+        error: null,
       };
     case ACTIONS.fetchFailed:
       return {
         ...state,
-        [payload.name]: {
```

Comment:
So is fido now instantiated per request type or something? How do you get rid of the name here without cancelling unrelated requests?

Reply:
The useFido hook now only worries about its singular request.  Having a single hook manage to multiple requests is really kind of an antipattern for hooks and added a bunch of needless complexity.  Each instantiation of the hook maintains its own state and ref to the abort controller, rather than requiring the hook to keep track of a list of requests.  Does that answer the question?  We could hop on voice and discuss it if you want to know more of my reasoning behind the change.

Reply:
Ah so we changed the way we're using it, that makes sense. 

I'm a huge fan of less tracked state, I just don't understand how hooks work yet :P 
Good change in my book.